client/lib/plans/constants.js

```diff
@@ -268,7 +267,14 @@ export const plansList = {
 	}
 };
 
-const allPaidPlans = [
+export const allPaidPlans = [
```

Comment:
Why? We are exporting `featuresList` and `plansList` as well without screaming snake case. :)